build.sh

```diff
@@ -79,6 +82,9 @@ set_defaults() {
     : ${BML_TESTING:=yes}
     : ${FORTRAN_FLAGS:=}
     : ${EXTRA_LINK_FLAGS:=}
+    : ${BML_GPU:=no}
+    : ${GPU_ARCH:=""}
```

Comment:
For consistency with above and you drop the `””`!

Reply:
Done